sdcm/nemesis.py

```diff
@@ -1687,6 +1688,7 @@ def disrupt_remove_service_level_while_load(self):
                                  name=SERVICE_LEVEL_NAME_TEMPLATE % (removed_shares, random.randint(0, 10)),
                                  shares=removed_shares).create())
 
+    @decorate_with_context(ignore_ycsb_connection_refused)
     def _deprecated_disrupt_stop_start(self):
```

Comment:
This is not the nemesis you wanted to decorate...

This one says deprecated in its name...

Reply:
You are absolutely right. This was a mistake. Changed to disrupt_stop_start_scylla_server